core/src/main/kotlin/entity/interaction/ActionInteraction.kt

```diff
@@ -228,218 +210,3 @@ public class ResolvedObjects(
-    supplier: EntitySupplier = kord.defaultSupplier
-): GlobalApplicationCommandInteraction {
-    return when (data.data.type.value) {
-        ApplicationCommandType.ChatInput -> GlobalChatInputCommandInteraction(data, kord, supplier)
-        ApplicationCommandType.User -> GlobalUserCommandInteraction(data, kord, supplier)
-        ApplicationCommandType.Message -> GlobalMessageCommandInteraction(data, kord, supplier)
-        is ApplicationCommandType.Unknown -> error("Unknown interaction.")
-        null -> error("No component type was provided")
-    }
-}
-
-/**
- * An [ActionInteraction] that took place in a Global Context with [dev.kord.core.entity.application.GuildApplicationCommand].
- */
-
-
-public sealed interface GuildApplicationCommandInteraction : ApplicationCommandInvocationInteraction, GuildInteractionBehavior {
-
-    override val guildId: Snowflake
-        get() = data.guildId.value!!
-
-    /**
-     * Overridden permissions of the interaction invoker in the channel.
-     */
-    public val permissions: Permissions get() = data.permissions.value!!
-
-
-    /**
-     * The invoker of the command as [MemberBehavior].
-     */
-    public val member: Member
-        get() = Member(
-            data.member.value!!,
-            data.user.value!!,
-            kord
-        )
-
-    override val channel: GuildMessageChannelBehavior
-        get() = GuildMessageChannelBehavior(guildId, channelId, kord)
-
-    override val user: UserBehavior
-        get() = UserBehavior(member.id, kord)
-
-    override fun withStrategy(strategy: EntitySupplyStrategy<*>): GuildApplicationCommandInteraction =
-        GuildApplicationCommandInteraction(data, kord, strategy.supply(kord))
-
-}
-
-public fun GuildApplicationCommandInteraction(
-    data: InteractionData,
-    kord: Kord,
-    supplier: EntitySupplier = kord.defaultSupplier
-): GuildApplicationCommandInteraction {
-    return when (data.data.type.value) {
-        ApplicationCommandType.ChatInput -> GuildChatInputCommandInteraction(data, kord, supplier)
-        ApplicationCommandType.User -> GuildUserCommandInteraction(data, kord, supplier)
-        ApplicationCommandType.Message -> GuildMessageCommandInteraction(data, kord, supplier)
-        is ApplicationCommandType.Unknown -> error("Unknown interaction.")
-        null -> error("No interaction type provided.")
-    }
-}
-
-
-public fun OptionValue<*>.user(): User = value as User
-
-
-public fun OptionValue<*>.channel(): ResolvedChannel = value as ResolvedChannel
-
-
-public fun OptionValue<*>.role(): Role = value as Role
-
-
-public fun OptionValue<*>.member(): Member = value as Member
-
-
-public fun OptionValue<*>.string(): String = value.toString()
-
-
-public fun OptionValue<*>.boolean(): Boolean = value as Boolean
-
-
-public fun OptionValue<*>.int(): Long = value as Long
-
-
-public fun OptionValue<*>.number(): Double = value as Double
-
-
-public fun OptionValue<*>.mentionable(): Entity {
-    return value as Entity
```

Comment:
Did you remove this on purpose? It was not moved to another file. If not you could also add `.attachment()`